### ISSUE TITLE
Doesn't compile on Manjaro

### ISSUE THREAD

**Issue by MarcMagneto** [ihaque/memtestCL#10]
I cannot get it to work with the following error message

> make -f Makefiles/Makefile.linux64 
> g++ -c -O2 -Wall -DLINUX -I/include -L-L/lib/x86_64 -o memtestCL_core.o memtestCL_core.cpp
> memtestCL_core.cpp: In constructor ‘memtestState::memtestState(cl_context, cl_device_id)’:
> memtestCL_core.cpp:87:70: warning: ‘_cl_command_queue* clCreateCommandQueue(cl_context, cl_device_id, cl_command_queue_properties, cl_int*)’ is deprecated [-Wdeprecated-declarations]
>    87 |     ctx(context), dev(device), cq(clCreateCommandQueue(ctx,dev,0,NULL)),
>       |                                                                      ^
> In file included from /usr/include/CL/opencl.h:47,
>                  from memtestCL_core.h:45,
>                  from memtestCL_core.cpp:14:
> /usr/include/CL/cl.h:1443:1: note: declared here
>  1443 | clCreateCommandQueue(cl_context                     /* context */,
>       | ^~~~~~~~~~~~~~~~~~~~
> memtestCL_core.cpp: In constructor ‘memtestFunctions::memtestFunctions(cl_context, cl_device_id, cl_command_queue)’:
> memtestCL_core.cpp:348:28: error: ‘memtestCL_kernels_len’ was not declared in this scope
>   348 |     size_t kernel_length = memtestCL_kernels_len;
>       |                            ^~~~~~~~~~~~~~~~~~~~~
> memtestCL_core.cpp:350:39: error: ‘memtestCL_kernels’ was not declared in this scope
>   350 |     const char* kernelcode = (char*) &memtestCL_kernels[0];
>       |                                       ^~~~~~~~~~~~~~~~~
> make: *** [Makefiles/Makefile.linux64:31: memtestCL_core.o] Error 1

**Comment by ihaque** [comment]
The error message suggests that `memtestCL_kernels.clh` did not get generated properly (https://github.com/ihaque/memtestCL/blob/master/Makefiles/Makefile.linux64#L25). Can you verify whether that is the case?

**Comment by MarcMagneto** [comment]
You are right. The file `memtestCL_kernels.clh` exists, but it is empty. I got to mention that my programming skills are nowhere near sufficient to solve this on my own.

**Comment by mirh** [comment]
Try to install amd's sdk
https://aur.archlinux.org/packages/amdapp-sdk/

**Comment by MarcMagneto** [comment]
I installed AMD's SDK as recommended and also deleted all git files and compiled again. Compilation was successfully this time, though I don't know what the original error source was. Thanks for your help.
